### ISSUE TITLE
Learn curriculum for _near_

### ISSUE THREAD

**Issue by gabbard** [isi-vista/adam#747]
This is the only relative preposition we plan to do, because of  #746

**Comment by gabbard** [comment]
After the curriculum is written, change the name to "Learn ..." and switch to the next milestone.

**Comment by gabbard** [comment]
The next task here is to record what our attribute learner does for _near_ and what representation it learns.

**Comment by LichMaster98** [comment]
See: https://github.com/isi-vista/adam/issues/836 for additional information about near. We want to make a modification to the language generator to use near as in the example "Mom pushes a ball near a box" as compared to "Mom goes to a box"

**Comment by paynesa** [comment]
[slot1_near_slot2.pdf](https://github.com/isi-vista/adam/files/4897275/slot1_near_slot2.pdf)

[learnerHTMLdump.pdf](https://github.com/isi-vista/adam/files/4897280/learnerHTMLdump.pdf)

Results for training the M9 subset relation learner on just "near" -- this seems to be working but should be checked and updated after #880 is complete.

**Comment by gabbard** [comment]
@paynesa : This looks good.  Can you close this issue and make a new one at lower priority for distinguishing two proximal object, one of which is more proximal than another?

**Comment by paynesa** [comment]
#887 covers the above.
